src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -143,9 +147,11 @@ public static CategoricalSample buildCategoricalSample(Map<String, List<String>>
             return dataSet;
         }
 
+
         /**
-         * Simple method that parses data set from a csv file
-         * without knowing the headers of the file
+         * Simple method that parses a
+         * data set from a csv file
```

Comment:
The way this is written means that there is no column info in the data set....this is very unlikely to happen...

Reply:
Yes it is unlikely, but it sometimes does occur. When the user will specify the headers or when the  . i came across this options that's why I included it. if you really think it's unnecessary I will remove it.